powershell/VismaDeveloperPortal/upload/README.md

```diff
@@ -108,17 +108,29 @@ Inside the **config.xml** file you will find these parameters:
 > Indicates if you want to store the logs in your machine.
 > 
 > Must be set to any of these values:  
-> **· false:** the logs will only be shown in the console.  
-> **· true:** the logs will be shown in the console and will be stored in your machine.
+> **� false:** the logs will only be shown in the console.  
```

Comment:
is this symbol the expected one? the previous version contains a dot (.)

Reply:
This was not expected. I updated the file.